TTPChallenge/QuizVC.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  QuizPartOneVC.swift
+//  TTPChallenge
+//
+//  Created by Luna An on 2/14/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class QuizVC: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    @IBOutlet weak var progressLabel: UILabel!
+    @IBOutlet weak var nextButton: UIButton!
+    @IBOutlet weak var tableView: UITableView!
+    
+    var objectsArray: [QuizGroup] = []
+    var allQuestions: [[Question]] = []
+    var responsesArray: [Int] = []
+    var techJobsForResult: [String] = []
+    var jobIndexArray: [Int] = []
+    var respondedQuestions = 0
+    
+    var qa = [String]()
+    var it = [String]()
+    var wd = [String]()
+    var md = [String]()
+    var da = [String]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        for _ in 1...4 {
+            qa.append(Answer.none)
+            it.append(Answer.none)
+            wd.append(Answer.none)
+            md.append(Answer.none)
+            da.append(Answer.none)
+        }
+        for (i,data) in Quizzes.personalityData.enumerated() {
+            var newQuestions: [Question] = []
+            
+            for string in data {
+                newQuestions.append(Question(string: string, section: Quizzes.techJobs[i]))
+            }
+            allQuestions.append(newQuestions)
+        }
+        tableView.separatorStyle = .none
+        
+        for i in 0..<5 {
+            let quizObject = QuizGroup(sectionName: Quizzes.techJobs[i], questions: allQuestions[i])
+            objectsArray.append(quizObject)
+        }
+        DispatchQueue.main.async {
+            self.activateResultButton()
+        }
```

Comment:
You are correct. I had used this method to update a progress bar that previously attached to the view - which has been removed. This method is longer needed!